nucypher/utilities/porter/control/specifications/porter_schema.py

```diff
@@ -102,6 +104,15 @@ def check_valid_quantity_and_include_ursulas(self, data, **kwargs):
         if ursulas_to_include and len(ursulas_to_include) > data['quantity']:
```

Comment:
Just my 2 cents - Maybe we could clarify a bit what `quantity` means here:
a) "Give me a `quantity` of Ursulas, and also include Ursulas from  the `include_ursulas`  list", for a total of `quantity` + `len(include_ursulas)` Ursulas
b) "Give me a maximum of `quantity` of Ursulas, with a preference for Ursulas in `include_ursulas`",  for a total of `quantity` Ursulas
If it's "b)" maybe we could go for `max_quantity` instead of `quantity`

Reply:
In this case it is more like b), except that if we use `max_quantity` it can imply that the returned number of ursulas has the option to be less than the max. It is more of a *"Give me `quantity` ursulas, with a preference for Ursulas in `include_ursulas"`*. 

Note that It isn't guaranteed that all of the Ursulas in `include_ursulas` will be included since they may have connectivity issues at the time of the operation, in which case they will not be included (tests don't have this problem 😄 )

The current terms used in the python API are `n` and `handpicked_ursulas` for a grant, but felt too ambiguous for the REST API.